docs/utilities.md

```diff
@@ -1400,4 +1400,43 @@ One final important variable is ```.grafana.del```, this dictates the delimeter
 between options in the drop down menus. This has significant repercussions if 
 one of your columns includes full stops, eg. email adresses. As a result we have 
 left this as definable so that the user can alter this to a non-disruptive value 
-for their data eg./.
\ No newline at end of file
+for their data eg./.
+
+## cloudutils.q
```

Comment:
I think this should be further up, before the stuff about API table etc.

Reply:
Sorry just to be clear do you mean move the whole section to just before the Full API section, or add the new namespace/ fns to the Full API list and API table or both?

Reply:
The first